src/share_commit.rs

```diff
@@ -96,20 +96,19 @@ fn next_lower_power_of_2(num: usize) -> usize {
 }
 
 mod nmt {
-    use nmt_rs::{CelestiaNmt, NamespaceId};
-    /// Build an nmt from leaves that are already prefixed with their namespace
-    pub fn build_nmt_from_namespaced_leaves(namespaced_leaves: &[impl AsRef<[u8]>]) -> [u8; 48] {
-        let mut tree = CelestiaNmt::new();
-        for leaf in namespaced_leaves.iter() {
-            let namespace: NamespaceId = leaf.as_ref()[..8]
-                .as_ref()
-                .try_into()
-                .expect("Namespace length is correct");
-            tree.push_leaf(&leaf.as_ref()[8..], namespace)
-                .expect("Leaves are pushed in order");
-        }
-        tree.root().0
-    }
+    // /// Build an nmt from leaves that are already prefixed with their namespace
```

Comment:
Nit: Should we remove it instead of commenting it out?

Reply:
Going to wait on this one pending Celestia changes to surface `nmt` proofs which might make this method required again